examples/invoice-parser/model/model.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, List
+
+from .parser_utils import extract_pdf_data
+
+
+
+class InvoiceParserModel(object):
```

Comment:
Would be good to leave a doc comment here explaining that the model does not work out of the box, which secret needs to be provided and how to obtain that secret.